src/rasenmaeher_api/web/api/enrollment/views.py

```diff
@@ -785,7 +841,7 @@ async def post_invite_code(
 
     # Random string for invite-code eg. GLXBT0
     _invite_code = "".join(random.choices(string.ascii_uppercase + string.digits, k=8))  # nosec B311
-
+    print(_invite_code)
```

Comment:
Never print, always use LOGGER.debug check https://github.com/pvarki/python-rasenmaeher-api/blob/main/src/rasenmaeher_api/jwtinit.py for reminder on standard LOGGER init

Reply:
👍